translations/cn.js

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This should be an object for easier reference

chinesetranslate: { <englishword>: <chinese_equiv> }

var chinesetranslate: { 'hello': 'Nihao'}

}